src/main/java/com/alkemy/ong/model/News.java

```diff
@@ -0,0 +1,53 @@
+package com.alkemy.ong.model;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+@Entity
+@SQLDelete(sql = "UPDATE news SET is_on = false WHERE new_id =?")
+@Where(clause = "is_on = true")
+@Getter @Setter
+@NoArgsConstructor @AllArgsConstructor
+@ToString
+public class News {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "new_id")
+    private Long id;
+
+    @Column(nullable = false)
+    @NotNull(message = "Name can not be empty.")
+    @NotEmpty(message = "Name can not be empty.")
+    @NotBlank(message = "Name can not be blank.")
```

Comment:
En este caso (y los demas) te alcanza con utilizar simplemente el @NotBlank, ya que alli ya estas evaluando la nulabilidad antes, es decir, @NotBlank devolvera true si NO es nulo y ademas la longitud del String sin contemplar los espacios adelante y atras es mayor a 0.
Te dejo un link para que veas las diferencias: https://stackoverflow.com/questions/17137307/in-hibernate-validator-4-1-what-is-the-difference-between-notnull-notempty